tap_zendesk_chat/__init__.py

```diff
@@ -76,5 +76,12 @@ def main():
         ctx = Context(args.config, args.state, catalog)
         sync(ctx)
 
+def main():
+    try:
+        main_impl()
+    except Exception as exc:
+        LOGGER.critical(exc)
```

Comment:
I think the proper way to log this would be:

```python
LOGGER.critical("unknown error in tap", exc_info=exc)
```

Reply:
I actually want the message from the exception to be on the CRITICAL line. When we run taps at Stitch we now scan the logs for CRITICAL lines and use those messages in notifications for the end user. If I change the message as you suggested, the output would look like:

```
CRITICAL unknown error in tap
```

Reply:
I see. This won't actually log the stacktrace though right? But maybe you don't care since you're re-raising?

Reply:
That's right. That's exactly why we're re-raising.

For some taps, we've changed it so they raise specific exceptions, and if we catch one of those exceptions at the top level, we log at the CRITICAL level but exit(1) rather than re-raise, to avoid printing the stack trace. We'd only do that if we had a good understanding of why an exception was occurring and would not need a stack trace. For example: https://github.com/singer-io/tap-facebook/pull/23/files